pkg/ddc/alluxio/transform_optimization.go

```diff
@@ -66,7 +66,11 @@ func (e *AlluxioEngine) optimizeDefaultProperties(runtime *datav1alpha1.AlluxioR
 	setDefaultProperties(runtime, value, "alluxio.user.file.passive.cache.enabled", "false")
 	setDefaultProperties(runtime, value, "alluxio.user.block.avoid.eviction.policy.reserved.size.bytes", "2GB")
 	setDefaultProperties(runtime, value, "alluxio.master.journal.folder", "/journal")
-	setDefaultProperties(runtime, value, "alluxio.master.journal.type", "UFS")
+	if value.Master.Replicas > 1 {
```

Comment:
If possible, could you please add unit test for this change? Thanks.

Reply:
> If possible, could you please add unit test for this change? Thanks.

I don't know which file should the unit test write to, could you tell me?